Dockerfile-acceptance

```diff
@@ -1,7 +1,18 @@
-FROM ruby:2.2
+FROM node:5
 
-COPY acceptance_tests /code
-WORKDIR /code
-RUN bundle install --deployment
+WORKDIR /test
 
-ENTRYPOINT "cucumber"
+ENV PHANTOM_JS phantomjs-2.1.1-linux-x86_64
+
+RUN wget https://github.com/Medium/phantomjs/releases/download/v2.1.1/${PHANTOM_JS}.tar.bz2 && \
```

Comment:
i still need to get that running as another container don't like that we have to do this